contrib/devtools/lint-python-shebang.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+for PYTHON_FILE in $(git ls-files -- "*.py")
+do
+    if [ "$(head -n 1 $PYTHON_FILE)" != "#!/usr/bin/env python3" ]
```

Comment:
`shellcheck` says:

```
    if [ "$(head -n 1 $PYTHON_FILE)" != "#!/usr/bin/env python3" ]
                      ^-- SC2086: Double quote to prevent globbing and word splitting.
```

My suggested formulation:

```
    if [[ $(head -n 1 "$PYTHON_FILE") != "#!/usr/bin/env python3" ]]; then
        …
    fi
```